Documentation/frequently-asked-questions.md

```diff
@@ -0,0 +1,121 @@
+
+Changing the privacy status to optedOut will clear the ECID value used by the Identity for Experience Cloud ID Service extension. Changing the privacy status back to optedIn will generate a new ECID used by the Identity for Experience Cloud ID Service extension. Privacy status changes do not change the ECID used by the Identity for Edge Network extension. Changing the privacy status will cause the ECID used by each identity extension to be different.
+
+When each identity extension has a different ECID, the Identity for Edge Network extension will include the Identity for Experience Cloud ID Service ECID in its [IdentityMap](api-reference.md), and so the Adobe Experience Platform Identity Service will link the the two ECIDs in the customer's Identity Graph.
+
+The following example shows an IdentityMap containing the ECIDs from both Identity for Edge Network extension and Identity for Experience Cloud ID Service extension. The ECID from the Identity for Edge Network extension is always listed first in the list of ECIDs.
+
+```json
+  "identityMap" : {
+      "ECID" : [
+        {
+          "id" : "73586628797489658169123381027155647197",
+          "authenticatedState" : "ambiguous",
+          "primary" : false
+        },
+        {
+          "id" : "81117527655405132265917409409236407340",
+          "authenticatedState" : "ambiguous",
+          "primary" : false
+        }
+      ]
+    }
+```
+### Q: I set privacy status to opted out, why do I see an ECID value when calling Identity.getExperienceCloudId()?
+
+A: The Identity for Edge Network extension does not change its ECID based on privacy status changes.
+
+**Note** 
+The Identity for Edge Network extension and the Identity for Experience Cloud ID Service extension each manage their own ECID value and are generated independently of each other.
+
+The Identity for Edge Network extension does not clear its stored identities or regenerate the ECID due to privacy status changes. Instead, use the [MobileCore.resetIdentities](https://github.com/adobe/aepsdk-core-android/blob/main/Documentation/Usage/MobileCore.md) API. Note this API does not clear the ECID but instead generates a new ECID.
+
+Each identity extension has its own API to retrieve their respective ECIDs as well. Use [Identity.getExperienceCloudId](https://github.com/adobe/aepsdk-edge-android/blob/main/Documentation/api-reference.md) to get the Identity for Edge Network extension's ECID, and [Identity.getExperienceCloudId](https://github.com/adobe/aepsdk-core-android/blob/main/Documentation/Usage/Identity.md) to get the Identity for Experience Cloud ID Service extension's ECID.
```

Comment:
the first link to Identity.getExperienceCloudId should point to this repo api-reference.md